UnicodeJsps/src/test/java/org/unicode/jsptest/TestUnicodeSet.java

```diff
@@ -161,7 +161,7 @@ public void TestIdentityQuery() {
 
     @Test
     public void TestNullQuery() {
-        checkSetsEqual("\\p{Bidi_Paired_Bracket=@none@}", "\\p{Bidi_Paired_Bracket_Type=None}");
+        checkSetsEqual("\\p{Bidi_Paired_Bracket=@none@}", "\\p{Bidi_Paired_Bracket_Type=Is_None}");
```

Comment:
It makes sense to me that a *binary property* would be prefixed with "is", but I wouldn't expect anything else -- non-binary property, or some value -- to work like that. Seems like gratuitous, confusing flexibility.

Reply:
> Seems like gratuitous, confusing flexibility.

I agree. But I think you need to travel to Redmond and to the year 2009 to make that case…

I suppose for the implicit properties script and gc, \p{isGreek} or \p{isLetter} still looks vaguely nice…